packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-review/app-review.html

```diff
@@ -159,6 +109,60 @@
                     </span>
                 </oui-tile-description>
             </oui-tile-definition>
+            <oui-tile-definition
+                data-term="{{:: 'pci_app_add_review_price_resource' | translate}}"
+            >
+                <oui-tile-description>
+                    <ovh-manager-catalog-price
+                        data-ng-if="$ctrl.resourcePriceInUcents > 0"
+                        data-price="$ctrl.resourcePriceInUcents"
+                        data-tax="$ctrl.resourcePriceTax"
+                        data-user="$ctrl.user"
+                        data-show-zero-price="true"
+                        data-interval="'hour'"
+                    >
+                    </ovh-manager-catalog-price>
+                    <p
+                        ng-if="$ctrl.resourcePriceInUcents === 0"
```

Comment:
```suggestion
                        data-ng-if="$ctrl.resourcePriceInUcents === 0"
```

Reply:
I guess it's code that's been moved and had this issue but it would be better if we could fix that :)